test/mocha/helpers.js

```diff
@@ -22,7 +25,7 @@ exports.createKeystore = async ({capabilityAgent, referenceId}) => {
   if(referenceId) {
     config.referenceId = referenceId;
```

Comment:
@dlongley @gannan08 Are `referenceId` at thing we're using now and want to continue using into the future?  If so, some of the validation we've added for keystores elsewhere will need to be revised.  I did not know this property was floating around as it does not appear anywhere else in our test suites.

Reply:
We're using it now -- as for the future, unknown. So, yes, we'll need validation for it.

Reply:
This is now allowed in bedrock-kms.